task 16.4

```diff
@@ -0,0 +1,13 @@
+SELECT I. *, IL. NAME
+FROM ISSUES I
+JOIN ISSUESLISTS IL ON I.ISSUESLIST_ID = IL.ID;
+
+SELECT I. *, U. FIRSTNAME, U.LASTNAME
+FROM USERS U
+JOIN ISSUES I ON U.ID = I.USER_ID_ASSIGNEDTO;
```

Comment:
Chociaż nie ma to znaczenia pod względem otrzymanego wyniku, bardziej zgodne z treścią zadania byłoby złączenie `ISSUES JOIN USERS` zamiast obecnego `USERS JOIN ISSUES`.